### ISSUE TITLE
gint ice mask 

### ISSUE THREAD

**Issue by benoitp-cmc** [NOAA-EMC/WW3#316]
In ww3_gint, forcings parameters (winds, ice, etc) are masked where ice concentration is greater than CICEN. 

This is not the case for output directly out of ww3_shel, where we get wind and ice output even for points where ice concentration is greater than CICEN. I would like to see this behaviour in gint.

This is with IC0.

**Comment by benoitp-cmc** [comment]
I'll do a new pull request for this after #794 is resolved. 

Rather than change the definition of `ACTIVE` as in #743, the check on `ACTIVE` should be removed for all parameters in the first family.
